### PR TITLE
Amend HIP19 template to add more detailed customer support and dashboard requirements

### PR DESCRIPTION
**Pull request by cokes518**
We propose today an amendment to HIP-19 to expand support requirements of Makers on the Helium network.

Today, Helium Inc supplies the firmware image and the OTA capabilities for Helium Hotspots and RAK Hotspot Miners. And some makers approved to date are also planning on developing their own Management Dashboard solution to accomplish the same (some with additional features). Bobcat, Syncrob.it, and Nebra all are planning to ship their respective Hotspots with the firmware support.

To ensure customer success and to maintain stability of the Helium Network (meaning the entire fleet has the latest updates), we’d like to amend HIP-19 to include supporting Hotspot-firmware update management, whether it is a Maker-supplied support dashboard, or a compatible Hotspot fleet management dashboard.

When submitting the proposal, please include a new section titled “Hotspot Fleet management plan” and outline how you plan on supporting Hotspot firmware updates, deployment, and rollback.

In addition, please include how you plan on supporting the ongoing e-commerce store and community feedback in Discord.

Makers are required to maintain and staff a Discord channel to communicate with the community, respond in a timely manner (1 business day), able to provide detailed updates during pre-order period, and communicate in an official capacity on manufacturing status (blog post, website, store page are examples of “official” outlets).

Ed: 
* rendered view of complete new template: https://github.com/helium/HIP/blob/ct/hip-19-amendment/0019-third-party-manufacturers/TEMPLATE.md
* diff https://github.com/helium/HIP/pull/141/files

### CODE REVIEW

**Comment by jamiew** [comment]
@cokes518 I pushed a few changes to try and clarify some of the language, including sticking with "manufacturer" instead of "maker" which isn't used anywhere else: https://github.com/helium/HIP/compare/7ae9eaa...dc94d62

I didn't understand the dashboard requirements you mention though. What exactly are you expecting? Are there any examples that could be referenced?

**Comment by ganey** [comment]
> I didn't understand the dashboard requirements you mention though. What exactly are you expecting? Are there any examples that could be referenced?

Personally I think access to some logs in the dashboard/device debugger might be rather useful, then technically abled members of the community might be able to provide assistance in a manufacturers absense, it may also help manufacturers provide better support without having to get customers to return devices.

For company controlled OS images, a rough guide on how a customer could take control of their own updates (at their own risk) would be useful, especially if a company decides to no longer support an old miner, or the company ceases trading.

**Comment by cvolkernick** [comment]
Is it possible to use some sort of open-sourced "kernel" (for lack of a better term/analogy coming to mind) that includes all of the core, commodified support needs (OTA support, remote management, etc.), and then each manufacturer can implement proprietary extensions on top of said core features if they so choose?

I get the need to differentiate from competitors, however it seems that once you get to the point where you are fundamentally dependent on a particular vendor's ongoing support, you are opening up the possibility (and likely inevitability) that a given vendor goes out of business for any arbitrary reason (lawsuits, mismanagement, personal reasons) and suddenly you have thousands if not tens of thousands of devices that will just become bricked due to no fault whatsoever of the customer / end user. If the core support needs (things absolutely required for the gateway to function properly) are standardized and open, it would allow for the development of a whole new market of third party support both of the firmware itself and repairs/maintenance.

TL;DR it seems like hardware support could perhaps be abstracted away from manufacturers in some capacity, so support isn't so dependent on them in the first place. I could be mistaken or misunderstanding key factors here, so open to corrections.

**Comment by shawaj** [comment]
> Is it possible to use some sort of open-sourced "kernel" (for lack of a better term/analogy coming to mind) that includes all of the core, commodified support needs (OTA support, remote management, etc.), and then each manufacturer can implement proprietary extensions on top of said core features if they so choose?
> 
> I get the need to differentiate from competitors, however it seems that once you get to the point where you are fundamentally dependent on a particular vendor's ongoing support, you are opening up the possibility (and likely inevitability) that a given vendor goes out of business for any arbitrary reason (lawsuits, mismanagement, personal reasons) and suddenly you have thousands if not tens of thousands of devices that will just become bricked due to no fault whatsoever of the customer / end user.
> 
> If the core support needs (things absolutely required for the gateway to function properly) are standardized and open, it would allow for the development of a whole new market of third party support both of the firmware itself and repairs/maintenance.
> 
> TL;DR it seems like hardware support could perhaps be abstracted away from manufacturers in some capacity, so support isn't so dependent on them in the first place. I could be mistaken or misunderstanding key factors here, so open to corrections.

@cvolkernick for reference this comment is somewhat related to your discussion points here https://github.com/helium/HIP/pull/268#issuecomment-903298140

**Comment by jamiew** [comment]
Closing stale issue
